.gitignore

```diff
@@ -2,6 +2,8 @@
 ### GitHub repository .gitignore section
 ###
 
+composer.lock
```

Comment:
Makes perfect sense and you are correct. I'll commit it back. Thanks for giving me some more insight into that @rbayliss